image/db/rhel/Dockerfile

```diff
@@ -25,18 +25,20 @@ COPY --from=extracted_bundle /bundle/etc/postgresql.conf /bundle/etc/pg_hba.conf
 
 ARG POSTGRESQL_ARCH=x86_64
 
-RUN groupadd -g 70 postgres && \
+RUN microdnf install -y shadow-utils && \
+    groupadd -g 70 postgres && \
     adduser postgres -u 70 -g 70 -d /var/lib/postgresql -s /bin/sh && \
-    dnf upgrade -y && \
-    dnf install -y \
+    curl https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-${POSTGRESQL_ARCH}/pgdg-redhat-repo-latest.noarch.rpm -o /tmp/pg_repo.rpm && \
+    rpm -i /tmp/pg_repo.rpm && \
```

Comment:
This step failed for me when I tried running this locally:

```
[root@beb09ad19088 /]# rpm -i /tmp/pg_repo.rpm
error: open of <html> failed: No such file or directory
error: open of <head><title>404 failed: No such file or directory
error: open of Not failed: No such file or directory
error: open of Found</title></head> failed: No such file or directory
error: open of <body> failed: No such file or directory
error: open of <center><h1>404 failed: No such file or directory
error: open of Not failed: No such file or directory
error: open of Found</h1></center> failed: No such file or directory
error: open of <hr><center>nginx</center> failed: No such file or directory
error: open of </body> failed: No such file or directory
error: open of </html> failed: No such file or directory
```

Reply:
```
[root@beb09ad19088 /]# ls -l /tmp/pg_repo.rpm 
-rw-r--r-- 1 root root 146 Oct  4 22:02 /tmp/pg_repo.rpm
```

Reply:
I feel like we tried this before and ran into the same issue, but I cannot find any PRs related to this at the moment

Reply:
It's working for me
```
docker run -it registry.access.redhat.com/ubi8/ubi-minimal:8.6 /bin/bash
[root@a7f6fa9ee662 /]# export POSTGRESQL_ARCH=x86_64
[root@a7f6fa9ee662 /]# curl https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-${POSTGRESQL_ARCH}/pgdg-redhat-repo-latest.noarch.rpm -o /tmp/pg_repo.rpm
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 13168  100 13168    0     0  24475      0 --:--:-- --:--:-- --:--:-- 24475
[root@a7f6fa9ee662 /]# rpm -i /tmp/pg_repo.rpm
warning: /tmp/pg_repo.rpm: Header V4 DSA/SHA1 Signature, key ID 442df0f8: NOKEY
[root@a7f6fa9ee662 /]# echo $?
0
```

Reply:
prow successfully build it https://quay.io/repository/stackrox-io/scanner-db/manifest/sha256:7da73ef2b03b0c4e713664c35c33f41317ff90763b60572a919409b3d84c097e?tab=layers

Reply:
You're right, I missed the ARCH :(